src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -167,36 +169,99 @@ function DeploymentConfigForm({ respondOnSuccess }) {
-                        getOptionLabel={option => `${option.version}`}
-                        getOptionValue={option => `${option.id}`}
-                        value={selectedChart}
-                        components={{
-                            IndicatorSeparator: null
-                        }}
-                        styles={{
-                            control: (base, state) => ({
-                                ...base,
-                                boxShadow: 'none',
-                                border: `solid 1px var(--B500)`
-                            }),
-                            option: (base, state) => {
-                                return ({
+                <div  style={{ display: 'grid', gridTemplateColumns: '1fr 1fr', gridColumnGap: '16px', marginBottom: '4px' }}>
+                    <div className="flex left column">
+                        <label className="form__label">Chart type</label>
+                    { isUnSet ? (
+                        <>
+                            <ReactSelect options={chartNames}
+                            isMulti={false}
+                            getOptionLabel={option => `${option.name}`}
+                            getOptionValue={option => `${option.name}`}
+                            value={selectedChart}
+                            components={{
+                                IndicatorSeparator: null
+                            }}
+                            styles={{
+                                control: (base, state) => ({
                                     ...base,
-                                    color: 'var(--N900)',
-                                    backgroundColor: state.isFocused ? 'var(--N100)' : 'white',
-                                })
-                            },
-                        }}
-                        onChange={(selected) => selectChart(selected as { id: number, version: string })}
-                    />
+                                    boxShadow: 'none',
+                                    border: `solid 1px var(--B500)`
+                                }),
+                                option: (base, state) => {
+                                    return ({
+                                        ...base,
+                                        color: 'var(--N900)',
+                                        backgroundColor: state.isFocused ? 'var(--N100)' : 'white',
+                                    })
+                                },
+                                container: (base, state) => {
+                                    return ({
+                                        ...base,
+                                        width: '100%'
+                                    })
+                                },
+                            }}
+                            onChange={(selected) => selectChart(selected as { id: number, version: string, name: string })}
```

Comment:
On changing, we are selecting the chart. which is wrong IMO.

Reply:
fixed the logic